gallery/src/GalleryNav/GalleryNav.tsx

```diff
@@ -21,43 +21,75 @@ import {
   complement,
   values
 } from 'ramda';
-import { NxTreeView, NxTreeViewChild, NxFilterInput, NxTreeViewProps } from '@sonatype/react-shared-components';
+import {
+  NxCollapsibleItems,
+  NxFilterInput,
+  NxCollapsibleItemsProps,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
 
-import pageConfig from './pageConfig';
+import pageConfig from '../pageConfig';
 import { useLocation } from 'react-router';
 import { NavLink } from 'react-router-dom';
-import { PageMapping, PageConfig } from './pageConfigTypes';
-import { markByFilter, matchesFilter } from './filterUtil';
-
-const renderLinks = (filter?: string) => pipe(
-    keys,
-    map((pageName: string) =>
-      <NxTreeViewChild key={pageName}>
-        <NavLink to={`/pages/${pageName}`} activeClassName="selected">
-          {markByFilter(filter, pageName)}
-        </NavLink>
-      </NxTreeViewChild>
-    )
-);
+import { PageMapping, PageConfig, PageType } from '../pageConfigTypes';
+import { markByFilter, matchesFilter } from '../filterUtil';
+
+import { faFile, faCode, faRulerCombined, IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import { faReact, faSass, faJs } from '@fortawesome/free-brands-svg-icons';
+
+import './GalleryNav.scss';
+
+const PAGE_TYPE_TO_ICON_MAP: { [page in PageType]: IconDefinition } = {
+  'documentation': faFile,
+  'react': faReact,
+  'html': faCode,
+  'layout': faRulerCombined,
+  'sass': faSass,
+  'js': faJs
+};
+
+const renderLinks = (filter?: string) =>
+  (entries: PageMapping) => Object
+      .entries(entries)
+      .map(([pageName, entry]) =>
+        <NxCollapsibleItems.Child key={pageName}>
+          <NavLink className="gallery-nav-link"
+                   to={`/pages/${pageName}`}
+                   activeClassName="selected">
+            {
+              entry
```

Comment:
This'll always be truthy though, right?

Reply:
> This'll always be truthy though, right?

Removed it here! ee35efc09fa75bdc70e7df7de160a2ab33edf201